### ISSUE TITLE
Perform site audit of mobile navigation

### ISSUE THREAD

**Issue by kristine-eudey** [hackforla/website#2190]
### Overview
We need to audit all the current mobile navigation on the site and create documentation for them in the design system so that the design team is able to pull from these options and design future mobile pages with greater ease and consistency.

### Action Items

- [x] Create new page in Figma titled "Mobile Menus Audit"
- [x] Go through all mobile pages in Figma, copying all instances of menus and navigation patterns used across the site.
- [x] Copy each instance to the "Mobile Menus Audit" page, noting which page on the site it came from
- [x]  Analyze findings: did you notice any inconsistencies or issues? Do you see any room for improvement, or recommend any changes based on what you found?
- [x] Present to the larger team to share the findings. 

### Resources/Instructions
[Hack for LA - Figma Mobile audit](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15122%3A95165)
[Basic patterns for mobile navigation: Pros and Cons (Smashing Magazine)](https://www.smashingmagazine.com/2017/05/basic-patterns-mobile-navigation/)

**Comment by kristine-eudey** [comment]
@ashleylin1 thanks for jumping in on this! Please provide an update on this issue:

- Progress
- Blockers
- Availability
- ETA

**Comment by kristine-eudey** [comment]
This looks great, thanks for compiling everything and identifying all the nav elements used, that was a ton of pages to look through! Just one thing left, which is to extract the nav elements you identified and copy them to create a sort of menu, similar to the way things are organized within the [design system](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3). 

The goal of this audit is for someone to be able to look in one place and be able to see all the design components and patterns related to navigation currently used on the site. This will allow designers to see the currently used options at a glance, so when designing new pages they can just copy and paste things, or if something is needed that isn't there, they can create it and add it to the system. 

**To summarize:** 
- Create a nav component menu - It should show one example of each type (hamburger, expandable card, dropdown, etc).
- Please pull things both from the current live pages, as well as mobile pages that have been designed but aren't yet live. 
- Please label next to each instance which pages the element is found on
- Place this menu on the same Figma page you're working on, putting a pink box around it when finished.

**Comment by ashleylin1** [comment]
Progress: Finished adding navigation component menu
Blockers: None
Availability: Weekends
ETA: Finished
Pictures: 
![image](https://user-images.githubusercontent.com/54569038/135549801-ba59a203-43e4-4f2f-869b-0c228bc193ad.png)
![image](https://user-images.githubusercontent.com/54569038/135549837-52c2c127-ed54-43be-bed2-1f60f92dc943.png)

**Comment by kristine-eudey** [comment]
Thank you @ashleylin1! Issue complete. Mobile navigation audit can be found here: [Figma - Mobile Navigation Audit](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15122%3A95165)

**Comment by ExperimentsInHonesty** [comment]
@ashleylin1 and @hackforla/website-design  Analyze  audit to determine findings and propose next steps

**Comment by kristine-eudey** [comment]
Moving issue back to prioritized backlog, ready for a new designer to take the issue from here and proceed with the analysis.

**Comment by IsaacDesigns** [comment]
Progress: Analyzing the mobile navigation audit and asking previous design lead, @kristine-eudey for clarification.
Blockers: Need more clarification for analysis and next steps.
Availability: 4 hours
ETA: Before the new year

**Comment by IsaacDesigns** [comment]
#### Issue status clarification:
I dont think the full audit should be added to the Design System, but it would be helpful to go through and identify if there's anything there that isn't represented in the Design System. Anything found that isn't represented in the Design System should then be assessed to determine if it should either be added to the DS, or changed to match something similar in the DS. 
-Kristine Eudey

**Comment by IsaacDesigns** [comment]
1. Progress: Will add the findings to the design system
2. Blockers: After looking at [Figma - Mobile Navigation Audit](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15122%3A95165) it seems like there was confusion as to what a mobile navigation menu is because all of the components listed are not navigation menus.
3. Availability: 4 hours
4. ETA: N/A

**Comment by IsaacDesigns** [comment]
Progress: I have determined the next steps in this issue is to identify if there's anything in this audit that isn't represented in the Design System. Anything found that isn't represented in the Design System should then be assessed to determine if it should either be added to the DS.
Blockers: None
Availability: 3 hours
ETA: 5 days

**Comment by ExperimentsInHonesty** [comment]
The navigation elements should be added to the design system if they are not there and if they are there, then the mobile views needed to be added as a context example.  In order to accommodate that, add two more columns
Desktop context example, Tablet, Mobile

**Comment by IsaacDesigns** [comment]
Progress: Bonnie and I have determined that the navigation elements should be added to the design system if they are not there and if they are there, then the mobile views needed to be added as a context example. In order to accommodate that I will add two more columns for Desktop, Tablet, and Mobile context example.
Blockers: None
Availability: 4 hours
ETA: 1/21/2022

**Comment by IsaacDesigns** [comment]
Finished adding components to Design System. We will need to create an issue to add two more columns for Desktop, and Tablet context examples.

**Comment by ExperimentsInHonesty** [comment]
I am moving this back to new issue approval column to indicate that the new design lead, or product team will have to make an issue for the changes needed to expand the design system.
